src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -232,6 +230,41 @@ private static void Run(string[] args)
         {
             Environment.ExitCode = app.Execute(args);
         }
+        catch (UnrecognizedCommandParsingException e)
+        {
+            string[] matches = e.NearestMatches.Take(3).ToArray();
+            string suggestion = matches switch
+            {
+                _ when matches.Length == 0 => "",
```

Comment:
Why not switch on `matches.Length` directly?

Reply:
yeah I knew I can do it simpler :D